src/styles/components/_action-block.scss

```
@@ -0,0 +1,5 @@
+.neeto-ui-action-block__wrapper {
+  display: flex;
+  align-items: center;
+  column-gap: 0.5rem;
```

@vijay2136 _a Let’s use gap.
```suggestion
  gap: 0.5rem;
```